public/data/Nonso.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Nonso Andrew Ugbodu",
+  "type": "personal",
+  "bio": "FrontEnd Engineer | Networking | Open Source Enthusiast!",
+  "avatar": "https://github.com/Arndy345.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Arndy345",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/nonso_andy",
+      "icon": "twitter"
+    },
+     {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/nonsougbodu/",
+      "icon": "linkedin"
+    },
+   "milestones": []
```

Comment:
There's a missing comma in here.
```suggestion
   "milestones": [],
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable